control-plane/roles/logging/tasks/main.yaml

```diff
@@ -12,19 +12,33 @@
         labels:
           name: "{{ logging_namespace }}"
 
-- name: Deploy loki-stack
+- name: Deploy basic auth secret for loki
+  k8s:
+    definition:
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: loki-basic-auth
+      type: Opaque
+      data:
+        auth: "{{ logging_ingress_loki_basic_auth | b64encode }}"
+    namespace: "{{ logging_namespace }}"
+    apply: true
+  when: logging_ingress_loki_basic_auth
+
+- name: Deploy loki
   include_role:
     name: ansible-common/roles/helm-chart
   vars:
     helm_repo: "{{ logging_chart_repo }}"
-    helm_chart: loki-stack
+    helm_chart: loki
     helm_target_namespace: "{{ logging_namespace }}"
-    helm_release_name: loki-stack
-    helm_chart_version: "{{ logging_chart_version }}"
-    helm_value_file_template: "loki-stack-values.yaml"
+    helm_release_name: loki
+    helm_chart_version: "4.6.1"
+    helm_value_file_template: "loki-values.yaml"
 
-- name: Create monitoring ingress for Loki
-  k8s:
-    definition: "{{ lookup('template', 'loki_ingress.yaml') }}"
-    namespace: "{{ logging_namespace }}"
-    apply: true
+# - name: Create monitoring ingress for Loki
+#   k8s:
+#     definition: "{{ lookup('template', 'loki_ingress.yaml') }}"
+#     namespace: "{{ logging_namespace }}"
+#     apply: true
```

Comment:
Can't remember if this is required or not, let's cleanup it up if not required and delete the template.